node_modules/col-tests/lib/util.js

```diff
@@ -18,12 +18,14 @@ var config = require('config');
 var randomstring = require('randomstring');
 var util = require('util');
 
+var AssetsTestUtil = require('col-assets/tests/util');
+var LtiTestUtil = require('col-lti/tests/util');
```

Comment:
It does, but for some reason it appears to be working fine. Do you want me to move `getAssetLibraryClient` back to the LTI module?